docs/authentication/session-management-overview.mdx

```diff
@@ -0,0 +1,132 @@
+- [Create Session Cookie](#where-to-store-tokens)
+- [Introspect a Token](/docs/next/validate-access-tokens)
+- [Revoke a Token](/docs/next/revoke-access-tokens)
+- [Retrieve Active Tokens](/docs/next/list-access-tokens)
+
+If you have enabled refresh tokens for your application, you can also refresh the access token to extend the duration of the user's session without requiring authentication.
+
+- [Refresh token request](/docs/next/enable-refresh-tokens)
+
+## How to store tokens
+
+Applications may utilize cookies, local storage or other strategies to associate session with the end-user’s application. Beyond Identity doesn’t provide an API/SDK for this purpose; it is up to you to decide how to handle the end-user’s session. One option is to store the JWT received from Beyond Identity in the end user’s application. Another solution is to store the JWT received from Beyond Identity on the server and use a cookie. This is a common solution offered by many web frameworks and is more secure as the JWT is more sensitive.
+
+### JWT Strategy with Refresh/Access Tokens
+
+The JWT (JSON Web Token) strategy with access/refresh tokens is a popular authentication and authorization mechanism used in web applications and APIs. It combines the benefits of JWTs with a token-based approach to manage user sessions securely. The strategy involves using two types of tokens: access tokens and refresh tokens.
+
+**Access Token:**
+
+- An access token is a compact and digitally signed JWT that contains information about the authenticated user and their permissions (claims).
+
+- It is short-lived and typically has a relatively short expiration time, usually just a few minutes or hours.
+
+- The access token is issued to a user after they successfully authenticate (e.g., login) and is used to grant access to protected resources or perform authorized actions on the server.
+
+- The server verifies the access token's authenticity and expiration to validate the user's identity and access privileges.
+
+**Refresh Token:**
+
+- A refresh token is also a JWT, but it is long-lived compared to the access token, often lasting days or even weeks.
+
+- The refresh token's purpose is to obtain a new access token once the previous one expires without requiring the user to log in again.
+
+- When the access token expires, the client can make a secure request to the server with the refresh token to obtain a new access token.
+
+- This helps improve user experience by allowing them to stay authenticated without frequent logins while also mitigating some security risks associated with long-lived tokens.
+
+The general flow of JWT strategy with access/refresh tokens works as follows:
+
+1. **Authentication:** When a user logs in with valid credentials, the server issues both an access token and a refresh token.
+
+1. **Accessing Protected Resources:** The client includes the access token in the request headers whenever accessing protected resources on the server.
```

Comment:
Here might be a good place to note and link to that section about supporting custom claims. Because they might want to check that when protecting a particular resource. 
https://developer-docs-git-sessionmanagement-beyondidentity.vercel.app/docs/next/create-api-token#example-create-a-token-with-custom-claims